templates/docs/upgrade-guide-v3.md

```diff
@@ -171,9 +171,9 @@ The `aria-label` attribute on table cells of the responsive table has been repla
 
 ## Chips
 
-By default, chips are not interactive anymore (they don't have hover, focus or active state). Apply the class to a button element to get interactive behaviour, e.g.: `<button class="p-chip">` instead of `<span class="p-chip">`.
+As chips are meant to be interactive by default they are now using `<button>` element. Any chips that don't have dismiss button should be built using `<button class="p-chip">` instead of `<span class="p-chip">`. Only the chips that have the dismiss button are still using span as their main element.
```

Comment:
```suggestion
As chips are meant to be interactive by default, they now use the `<button>` element. Any chips that don't have a dismiss button should be built using `<button class="p-chip">` instead of `<span class="p-chip">`. Only chips that have the dismiss button still use span as their main element.
```